pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -200,7 +200,7 @@ func (ku *KubeUtil) GetLocalPodList() ([]*Pod, error) {
 	}
 
 	// ensure we dont have nil pods
-	tmpSlice := make([]*Pod, 0, len(pods.Items))
+	tmpSlice := []*Pod{}
```

Comment:
Can you describe the bug that was occurring?  I don't quite see how this change is helping

Reply:
if `n := len(pods.Items)` `make([]*Pod, 0, len(pods.Items))` allocate `n` entry in the slice;
then `tmpSlice = append(tmpSlice, pod)` add the pod at the `tmpSlice[n]` position in the slice

Reply:
`make([]*Pod, 0, len(pods.Items))` will allocate a slice with `length=0` and `capacity=len(pods.Items)`.  Appending to the slice will work as it should

See https://play.golang.org/p/Gto3wKLGCfa for example

Reply:
right, I forgot this different between `length` and `cap`